lib/dfe_sign_in_api.rb

```diff
@@ -0,0 +1,35 @@
+module DFESignIn
+  class API
```

Comment:
Do we imagine moving the other DfESignIn methods into this class, or is this only for the `/users` endpoint?

Reply:
I believe it's open for extensions 🤔

Reply:
🤔